print_planner:
  setpoint_rate:            100 #Hz

  #Take-off and landing
  tol_speed:                0.1 #m/s
  tol_height:               0.5 #m
        
  #TOPPRA     
  print_vel:                0.2 #m/s
  print_max_accel:          1000000 #m/s^2
  
  transition_vel:           0.15 #m/s
  transition_max_accel:     0.05 #m/s^2
  
  max_yawrate:              10 #deg/s
  max_yawrate_dot:          10 #deg/s^2

  #Tolerance checking  
  start_speed_tolerance:    0.1 #m/s
  start_position_tolerance: 0.05 #m

  offset:                   [0, 0, 0] #[m, m, m]
        
  #Print progress      
  first_layer:              0